capa/main.py

```diff
@@ -500,7 +501,7 @@ def get_workspace(path, format_, sigpaths):
     else:
         raise ValueError("unexpected format: " + format_)
 
-    viv_utils.flirt.register_flirt_signature_analyzers(vw, sigpaths)
+    viv_utils.flirt.register_flirt_signature_analyzers(vw, [s.as_posix() for s in sigpaths])
```

Comment:
better str vs as_posix here?

Reply:
Will change to str representations as it will be used in runtime. Later when changes are made in viv utils package it will be changed to Path types